sdcm/nemesis.py

```diff
@@ -178,14 +178,6 @@ def disrupt_nodetool_drain(self):
         self._set_current_disruption('Drainer %s' % self.target_node)
         drain_cmd = 'nodetool -h localhost drain'
         result = self._run_nodetool(drain_cmd, self.target_node)
-        for node in self.cluster.nodes:
-            if node == self.target_node:
-                self.log.info('Status for target %s: %s', node,
-                              self._run_nodetool('nodetool status', node))
-            else:
-                self.log.info('Status for regular %s: %s', node,
-                              self._run_nodetool('nodetool status', node))
-
         if result is not None:
```

Comment:
Let's keep this nodetool status, because this is DURING the disruption vs the BEFORE and AFTER